### PR TITLE
Web3connect checkout

### PR DESCRIPTION
**Pull request by pedrouid**

### CODE REVIEW

File: pretix_eth/templates/pretix_eth/pending.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
-{% load i18n %}
-{% load static %}
-<script type="text/javascript" src="{% static "pretix_eth/check.js" %}"></script>
+{% load i18n %} {% load static %}
 <p>
-    <strong>{% blocktrans %}Deposit {{ amount }} {{ coin }} into {{ payment_info }}{% endblocktrans %}</strong>
+  <strong>
+    {% blocktrans %} Deposit {{ amount }} {{ coin }} into {{ payment_info }}
```
Comment (davesque) [inline pretix_eth/templates/pretix_eth/pending.html]: I think this should be `{% blocktrans trimmed %}` to ensure that modifying whitespace in the translation string doesn't cause any related translations not to be found.

**Comment by davesque** [comment]
LGTM. Nice and simple 👌.

File: pretix_eth/templates/pretix_eth/pending.html (hunk 1 above)
Comment (pedrouid) [reply pretix_eth/templates/pretix_eth/pending.html]: Done 👍

File: pretix_eth/templates/pretix_eth/checkout_payment_form.html [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
-      </div>
-    {% endif %}
-
-
-    {% bootstrap_form form layout='horizontal' %}
+  {% if ETH_address %}
+  <div class="ETH-crypto-payment-details">
+    <p>Send {{ ETH_per_ticket }} ETH to {{ ETH_address }}</p>
+    <a
+      href="https://checkout.web3connect.com/?currency=ETH&amount={{
```
Comment (ligi) [inline pretix_eth/templates/pretix_eth/checkout_payment_form.html]: does this need to be a 3rd party site or can it be a static site living here?

Comment (pedrouid) [reply pretix_eth/templates/pretix_eth/checkout_payment_form.html]: I don't see any problem being an external website just like a Paypal payment would happen. But feel free to fork and include it in the plugin if you have time.

Source code: https://github.com/pedrouid/web3connect-checkout

Comment (ligi) [reply pretix_eth/templates/pretix_eth/checkout_payment_form.html]: It adds a point of failure. In the case of paypal it is needed - I do not see the need here.

Comment (pedrouid) [reply pretix_eth/templates/pretix_eth/checkout_payment_form.html]: The problem was (as @pipermerriam described on Discord) the content-security policy of templates rendered by Django broke the JS logic which broke the libraries we used like Web3Connect

I will leave it up to you guys to include this PR in the first wave or not.

Comment (ligi) [reply pretix_eth/templates/pretix_eth/checkout_payment_form.html]: yea - this is for templates - but does this also apply for static sites? cc @pipermerriam

Comment (pipermerriam) [reply pretix_eth/templates/pretix_eth/checkout_payment_form.html]: I believe it's going to be pages served by django.

Comment (davesque) [reply pretix_eth/templates/pretix_eth/checkout_payment_form.html]: @ligi Django should be able to serve a static site.

**Comment by davesque** [comment]
@pedrouid Just a heads up, I rebased this PR onto the one I just made (#40 ) that fixes ambiguous payment instructions.

**Comment by ligi** [comment]
will be super-seeded by  #49

**Pull request closed without merging**